subprojects/docs/src/docs/userguide/reference/ci-systems/github-actions.adoc

```diff
@@ -0,0 +1,149 @@
+[[build_github_actions]]
+= Executing Gradle builds on GitHub Actions
+
+TIP: Top engineering teams using GitHub Actions have been able to reduce CI build time by up to 90% by using the Gradle Build Cache. https://gradle.com/training/build-cache-deep-dive/?bid=guides-execute-travisci[Register here] for our Build Cache training session to learn how your team can achieve similar results.
+
+Building Gradle projects doesn't stop with the developer's machine.
+https://en.wikipedia.org/wiki/Continuous_integration[Continuous Integration] (CI) has been a long-established practice for running a build for every single change committed to version control to tighten the feedback loop.
+
+In this guide, we'll discuss how to configure link:https://github.com/features/actions/[GitHub Actions] for a typical Gradle project.
+
+== What you'll need
+
+* A text editor
+* A command prompt
+* The Java Development Kit (JDK), version 1.8 or higher
+* A https://gradle.org/install[Gradle distribution], version {gradle-version} or better
```

Comment:
`{gradle-version}` is not expanded. It must be replaced with `{gradleVersion}`. It would be great to fix that in the other CI guides.

Reply:
Fixed in all guides